apps/AEPSampleApp/extensions/EdgeView.tsx

```diff
@@ -43,29 +102,42 @@ const EdgeView = ({navigation}: NavigationProps) => {
        locationStr = "null";
       }
       getlocationHintText(locationStr);
-  })
+    });
   }
 
   return (
     <View style={styles.container}>
       <ScrollView contentContainerStyle={{marginTop: 75}}>
         <Button onPress={() => navigation.goBack()} title="Go to main page" />
         <Text style={styles.welcome}>Edge v{version}</Text>
-        <Button title="sendEvent()" onPress={() => sendEvent()} />
+        <Button title="sendEvent() v1" onPress={() => sendEventV1()} />
```

Comment:
instead of sendEvent() v1 / v2, can we have something like:
sendEvent(params)
sendEvent(object)